server/rest/geom.py

```diff
@@ -36,16 +39,26 @@ def __init__(self):
         self.route('DELETE', (':geomId',), self.deleteGeom)
         self.route('GET', ('export', ':itemId',), self.exportKPF)
 
+    # The girder default serialization takes twice the time than this custom serializer. Since geom could be relatively big, it worth to use this custom serializer. 
```

Comment:
Pedantic: `s/twice the time than/twice the time as/` and `s/it worth/it is worthwhile/`. Also, line length.

Reply:
Thanks for the grammar correction